packages/app/src/app/pages/Dashboard/Components/UpgradeBanner/UpgradeBanner.tsx

```diff
@@ -225,14 +226,15 @@ export const UpgradeBanner: React.FC<UpgradeBannerProps> = ({ teamId }) => {
                     gridColumnEnd: 'span 4',
                   },
 
-                  [`@media screen and (min-width: ${WRAP_WIDTH + 1}px)`]: {
+                  [`@media screen and (min-width: ${WRAP_WIDTH}px)`]: {
                     gridColumnEnd: 'span 6',
                   },
                 }}
                 key={f.icon}
                 as="li"
+                gap={1}
```

Comment:
I can't commit to the branch from CSB it seems, but this gap is causing a TS error and doesn't really do anything so it can be removed!
```suggestion
```